frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupItem/groupItem.component.tsx

```diff
@@ -87,9 +85,7 @@ export const GroupItem = ({ group, color, opacity }: GroupProps) => {
 						<PrimaryTicketButton onClick={toggleShowGroup}>
 							{groupIsVisible ? (<ShowIcon />) : (<HideIcon />)}
 						</PrimaryTicketButton>
-						<PrimaryTicketButton onClick={editGroup}>
-							<EditIcon />
-						</PrimaryTicketButton>
+						<EditGroupButton defaultValues={{ color, opacity, ...group }} />
```

Comment:
I mention this because I saw you are adding the `isAdmin` disabling across the group components, Could you also add it for the `delete group` button, such that, when the user is not admin, they cannot delete the group?